docs/resources/rms_policy_organizational_assignment.md

```diff
@@ -0,0 +1,145 @@
+---
+subcategory: "Config"
+---
+
+# huaweicloud_rms_organizational_policy_assignment
+
+Using this resource to assign the organizational policy HuaweiCloud resources.
+
+## Example Usage
+
+### Assign a built-in policy
+
+```hcl
+variable "policy_assignment_name" {}
+
+data "huaweicloud_organizations_organization" "test" {}
+
+data "huaweicloud_rms_policy_definitions" "test" {
+  name = "access-keys-rotated"
+}
+
+resource "huaweicloud_rms_organizational_policy_assignment" "test" {
+  organization_id      = data.huaweicloud_organizations_organization.test.id
+  name                 = var.policy_assignment_name
+  description          = "The maximum number of days without rotation. Default 90."
+  policy_definition_id = try(data.huaweicloud_rms_policy_definitions.test.definitions[0].id, "")
+  period               = "TwentyFour_Hours"
+
+  parameters = {
+    maxAccessKeyAge = "\"90\""
+  }
+}
+```
+
+### Assign a custom policy
+
+```hcl
+variable "function_name" {}
+variable "policy_assignment_name" {}
+
+data "huaweicloud_organizations_organization" "test" {}
+
+resource "huaweicloud_fgs_function" "test" {
+  name                  = var.function_name
+  code_type             = "inline"
+  handler               = "index.handler"
+  runtime               = "Node.js10.16"
+  functiongraph_version = "v2"
+  app                   = "default"
+  enterprise_project_id = "0"
+  memory_size           = 128
+  timeout               = 3
+}
+
+resource "huaweicloud_rms_organizational_policy_assignment" "test" {
+  organization_id = data.huaweicloud_organizations_organization.test.id
+  name            = var.policy_assignment_name
+  description     = "This is a custom policy assignment."
+  function_urn    = "${huaweicloud_fgs_function.test.urn}:${huaweicloud_fgs_function.test.version}"
+  period          = "TwentyFour_Hours"
+
+  parameters = {
+    string_test = "\"string_value\""
+    array_test  = "[\"array_element\"]"
+    object_test = "{\"terraform_version\":\"1.xx.x\"}"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `organization_id` - (Required, String, ForceNew) Specifies the ID of the organization.  
+  Changing this parameter will create a new resource.
+
+* `name` - (Required, String, ForceNew) Specifies the name of the organizational policy assignment.  
+  The valid length is limited from `1` to `60`, only letters, digits, hyphens (-) and underscores (_) are allowed.  
+  Change this parameter will create a new resource.
+
+* `excluded_accounts` - (Optional, String) Specifies the excluded accounts of the organizational policy assignment.
```

Comment:
this parameter should be a list of string?

Reply:
fixed